docs/content/node/getting-started/application/index.md

```diff
@@ -67,9 +66,25 @@ Once the library's installed, create a new file in the `src` directory called
 `authorization.ts`. In this file, we'll write helper functions for
 initializing and accessing Oso:
 
-{{< literalInclude
```

Comment:
Would have rather used a `literalInclude` here but it did not work. The wrong lines would display, some doubled up. Maybe worth investigating.